src/components/Alert.jsx

```diff
@@ -1,6 +1,44 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
+import Note from './Note';
+import { SetFieldValues } from "../utilities/FormHelpers";
+
+
+
+export const getAlertMessage = (props) => {
+	let message = '';
+	const { hasPasswordReset } = props.values;
+	if (hasPasswordReset) {
+		message = <Note className='bc_alert alert-success'>{props.values.SignInPage.ResetPasswordAlert.replace('{email address}', props.history.location.state)}</Note>
+	}
+	else if (props.status) {
+		const { incorrectEmail, networkError } = props.status;
+		if (incorrectEmail) {
+			message = <Note icon='Nothing' className='bc_alert alert-warning'>{incorrectEmail ? incorrectEmail : null}</Note>
+		}
+		else {
+			message = <Note icon='Nothing' className='bc_alert alert-warning'>{networkError ? networkError : null}</Note>
+		}
+	}
+	return message;
```

Comment:
this can be simplified

```js
const { incorrectEmail, networkError } = props.status;
const message = incorrectEmail || networkError || null;
return <Note icon='Nothing' className='bc_alert alert-warning'>{message}</Note>
```

Reply:
if you make these changes you will not need `let message` on line 10.